packages/neuron-wallet/src/controllers/helpers.ts

```diff
@@ -1,22 +1,34 @@
 import Key from '../keys/key'
 import { ResponseCode } from '.'
+import { CatchControllerError } from '../utils/decorators'
+import i18n from '../utils/i18n'
 
 export enum HelpersMethod {
   GenerateMnemonic = 'generateMnemonic',
 }
+
+/**
+ * @class HelpersController
+ * @description handle messages from helpers channel
+ */
 class HelpersController {
-  public static generateMnemonic = (): Controller.Response<string> => {
+  /**
```

Comment:
I'm not so sure about this but these comments don't really add many values. The scope and function names already describe themselves well enough.

Reply:
removed comments on methods of helpers controller and networks controller, but left comments on the controller classes for reminding us the controllers are used to handle messages from channels, not network requests.
https://github.com/nervosnetwork/neuron/pull/367/commits/bf3d7ba826846953acd7a0e2a7546c79944cd26f